glossary.yml

```diff
@@ -5944,6 +5944,12 @@
     term: የሃይፐር ጽሑፍ ምልክት ማድረጊያ ቋንቋ
     def: >
       ለድር ገጾች ጥቅም ላይ የዋለው መደበኛው [ምልክት ማድረጊያ ቋንቋ](#markup_language) ኤችቲኤምኤል ነው [DOM](#dom) (ዲጂታል ነገር ሞዴል) በመጠቀም በማስታወስ ውስጥ ተወክሏል።
+  uk:
+    term: "мова розмітки гіпертексту (HyperText Markup Language)"
+    acronym: "HTML"
+    def: >
+      Стандартна [мова розмітки](#markup_language), що використовується для веб-сторінок. HTML
+      представлено у пам'яті за допомогою [DOM](#dom) (цифрова модель об'єктів).
```

Comment:
DOM = Document_Object_Model, see e.g. https://en.wikipedia.org/wiki/Document_Object_Model - this should be corrected in the English version too and reviewed by someone who works on the English text. 

Note that German text says it right: Dokumenten-Objekt-Modell, as well as https://glosario.carpentries.org/en/#dom

Reply:
yes, should I create PR to fix English version?

Reply:
@Herobread - yes please, or alternatively, just add a separate commit which fixes it to this PR